concepts/ranges/about.md

```diff
@@ -0,0 +1,125 @@
+# Ranges
+
+[Ranges][ranges] represents an interval between two values.
+The most common types that support ranges are `Int` and `String`.
+They can be used for many things like quickly creating a collection, slicing strings, checking if a value is in a range and iteration.
+They are created using the range operator `..` or `...` (inclusive and exclusive respectively).
+
+```ruby
+1..5 # => 1, 2, 3, 4, 5
+1...5 # => 1, 2, 3, 4
+```
```

Comment:
The return is not what this statement returns, as we can see.  Using IRB, since IRB "evaluated" syntax is being used here in the code example:

~~~suggestion

In IRB:
```ruby
>> (1..5)
=> 1..5
>> (1...5)
=> 1...5
```
~~~

What do you think?  The return is not like it would be with an Array.

Probably not in this section, but we can coerce this to an Array to show what is originally here:

```irb
>> (1..5).to_a
=> [1, 2, 3, 4, 5]
>> (1...5).to_a
=> [1, 2, 3, 4]
```